agent/src/main/java/com/microsoft/hydralab/agent/runner/TestRunDeviceOrchestrator.java

```diff
@@ -129,6 +129,23 @@ public String stopScreenRecorder(@NotNull TestRunDevice testRunDevice, @NotNull
         }
     }
 
+    public void releaseScreenRecorder(@NotNull TestRunDevice testRunDevice, @Nullable Logger logger) {
+        if (testRunDevice.getScreenRecorder() == null) {
+            return;
+        }
+
+        if (logger != null) {
+            logger.info("Releasing screen recorder");
+        }
+        if (testRunDevice instanceof TestRunDeviceCombo) {
+            ((TestRunDeviceCombo) testRunDevice).getDevices().forEach(testRunDevice1 -> {
```

Comment:
nit picky: testRunDevice1 -> testRunDevice

Reply:
The testRunDevice has already defined in this group: 
![image](https://github.com/microsoft/HydraLab/assets/30514480/2eac0273-a554-4bb7-b5c0-4840d162384e)